static/app/components/discover/quickContextCommitRow.tsx

```diff
@@ -0,0 +1,94 @@
+import styled from '@emotion/styled';
+
+import UserAvatar from 'sentry/components/avatar/userAvatar';
+import CommitLink from 'sentry/components/commitLink';
+import {PanelItem} from 'sentry/components/panels';
+import TextOverflow from 'sentry/components/textOverflow';
+import {t, tct} from 'sentry/locale';
+import ConfigStore from 'sentry/stores/configStore';
+import space from 'sentry/styles/space';
+import {Commit} from 'sentry/types';
+
+import {formatCommitMessage} from '../commitRow';
+import ExternalLink from '../links/externalLink';
+
+interface QuickContextCommitRowProps {
+  commit: Commit;
+  customAvatar?: React.ReactNode;
+}
+
+function QuickContextCommitRow({commit, customAvatar}: QuickContextCommitRowProps) {
+  const user = ConfigStore.get('user');
+  const isUser = user?.id === commit.author?.id;
+  const hasPullRequestURL = commit.pullRequest && commit.pullRequest.externalUrl;
+
+  return (
+    <StyledPanelItem key={commit.id} data-test-id="quick-context-commit-row">
+      {customAvatar
+        ? customAvatar
+        : commit.author && (
+            <div>
```

Comment:
Do we need this `div`? I removed it and I don't think it's changed the styling

Reply:
Will remove the div.